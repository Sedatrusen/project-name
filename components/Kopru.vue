<template>
  <div class="kutu">
    <ul>
      <li>Great for New Players</li>
      <li>Games</li>
      <li>Games</li>
      <li>Subscriptions</li>
      <li>Mounts</li>
      <li>Pets</li>
      <li>Toys</li>
      <li>Transmogs</li>
      <li>Bundles</li>
      <li>Game Services</li>
      <li>Gear</li>
      <li class="Group">
        Group by
        <Select v-model="selection">
          <Option key="Categories" value="Categories">Categories</Option>
        </Select>
      </li>
      <li>
        Sort by
        <Select v-model="selection">
          <Option key="Categories" value="Categories">Categories</Option>
        </Select>
      </li>
    </ul>
  </div>
</template>

<script>
export default {}
</script>

<style scoped>
.kutu {
  background-image: url('../assets/top.png');
  background-color: #01234a;
  display: flex;
}
ul {
  color: #fffc;
  margin: auto;
  display: flex;
  list-style-type: none; /* Remove bullets */
  justify-content: space-between;
}
ul li {
  font-size: 1.3rem;
  margin: 0.8rem;
}
ul li:hover {
  color: white;
}
select {
  color: white;
  background-color: #001732;
}
.Group {
  margin-left: 20rem;
}
</style>
