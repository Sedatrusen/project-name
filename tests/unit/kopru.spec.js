import { shallowMount } from '@vue/test-utils'
import Kopru from '@/components/Newkopru.vue'

describe('Kopru component', () => {
  it('Kopru componenti calisiyor', () => {
    const wrapper = shallowMount(Kopru, {
      propsData: {},
    })
    expect(wrapper.exists()).toBe(true)
    expect(wrapper).toMatchSnapshot()
  })
})
