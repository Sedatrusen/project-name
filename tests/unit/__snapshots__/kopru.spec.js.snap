// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Kopru component Kopru componenti calisiyor 1`] = `[Function]`;
